opba-auth-rest-api/src/main/resources/static/tpp_auth_api.yml

```diff
@@ -126,14 +145,25 @@ components:
           schema:
             type: string
         X-Request-ID:
-          $ref: '#/components/parameters/X-Request-ID'
+          $ref: "#/components/parameters/X-Request-ID"
```

Comment:
Double quotes process '\n' characters in string as if it would be new line, this is not what is desired in URL path references, please use `'` for quoting

Reply:
sorry, ALL refs are double quoted in the whole file. I dont see any single quote in whole file. It does not make sense to me to change single?

Reply:
You have changed the file format from single-quoted to double-quoted. Single quotes are preferable file format. On dev branch this file is single-quoted